jgrapht-core/src/test/java/org/jgrapht/alg/shortestpath/KDisjointShortestPathsTestCase.java

```diff
@@ -925,6 +927,76 @@ public void testStar()
         }
     }
 
+    /**
+     * A complex test case with the goal of finding the three shortest paths from vertex 1 to vertex 2 through the following weighted directed graph. Vertices are numbers in boxes, i.e., 1, 3, 4, 5, 6, 7, 8, 2.
+     * Weights are numbers close to an edge. Each edge has its origin to the left, and destination to the right
+     * The source is node 1. Sink is node 2.
+     * The weight of each edge is the unboxed number close the edge.
+     *
```

Comment:
Add //@formatter:on and off around this diagram, otherwise it will get destroyed next time we reformat our code.

Reply:
Thank you for the suggestion.

I added formatter:off/on around the diagram.